mods/ra/sequences/ships.yaml

```diff
@@ -6,6 +6,7 @@ ss:
 ca:
 	idle:
 		Facings: 16
+		InterpolatedFacings: 1024
```

Comment:
This seems to have slipped in from testing, and I agree it doesn't look that good on ships due to the angles their facings were rendered at, so this can be removed.

Reply:
Sure, I have removed it, though perhaps it could be included with a smaller number of facings, e.g. 32 or 64. I will leave it to others to test on their own though, rather than having it in the PR.

Reply:
One more thing, I noticed that for quantized facings such as turrets, the interpolated facings don't rotate at the exact time as the normal facings, as you can see below. I suspect this might be because QuantizeFacingsFromSprites() runs either before or after InterpolatedFacings are calculated, making them 1 frame off. I'm not sure how to fix this however, since I suspect delaying or speeding up the calculation of InterpolatedFacings is not straightforward (and could have side effects on the Mig and other aircraft that don't use QuantizeFacigsFromSprites()).

https://www.dropbox.com/s/zuw4asuvt7kg3lw/staggered_facings.mkv?dl=0